lib/options-handler.js

```diff
@@ -296,16 +296,17 @@ export class ClientOptionsHandler {
                 });
                 semverGroups[compiler.group].splice(index, 0, compiler);
             }
-            _.each(compiler, (_, propKey) => {
+            Object.keys(compiler).map(propKey => {
```

Comment:
I think you can just use `for (let propKey of Object.keys(compiler)) {}` here

Reply:
Thanks for the insights @partouf . Even I feel `for of` is more readable and consistent with other loops. Can I make this change for all the `map` introduced in this PR?

Reply:
Yes, definitely